### PR TITLE
Add a flag for enabling AWS_SECURITY_TOKEN

### PR DESCRIPTION
**Pull request by mateimicu**
AWS_SECURITY_TOKEN was dropped in favor of AWS_SESSION_TOKEN, currently
aws-vault sets both variables in a new shell that can cause problems.

I preserved the default behavior but put it under a flag

I looked at `exec_test.go` I was wondering if it makes sense to add a test there and how would you do it?

Addresses #495

### CODE REVIEW

**Comment by j0hnsmith** [comment]
Can you document it https://github.com/99designs/aws-vault/blob/master/USAGE.md#environment-variables

**Comment by mateimicu** [comment]
@j0hnsmith I am not sure what to document in that section, I don't see `AWS_SECURITY_TOKEN` or `AWS_SESSION_TOKEN` being documented there.

File: cli/exec.go [hunk 1]
```diff
@@ -45,6 +46,9 @@ func ConfigureExecCommand(app *kingpin.Application) {
 
 	cmd := app.Command("exec", "Executes a command with AWS credentials in the environment")
 
+	cmd.Flag("use-legacy-aws-secure-token", "Export also the deprecated AWS_SECURE_TOKEN alongside the standard, AWS_SESSION_TOKEN").
```
Comment (mtibben) [inline cli/exec.go]: Lets flip it for now, maybe `--disable-legacy-env-vars`?

**Comment by mtibben** [comment]
Closing for now, haven't heard of anyone else with issues and it's easy to unset in the shell if required

**Pull request closed without merging**